Flaunch/Gapless/details.json

```diff
@@ -0,0 +1,5 @@
+{
+	"name": "Gapless",
+	"description": "A gapless layout for those that want a \"big wall of games\" kind of look",
+	"creator_id": "686777849490702433"
```

Comment:
Deniz Onmuş#9921 must have logged in at least once on Themezer before I can add the theme